pure_pursuit_hakuroukun:
  max_speed: 0.35            # Maximum speed [m/s]
  min_speed: 0.25            # Miximum speed [m/s]
  lookahead_distance: 1.1   # Lookahead distance [m]
  wheelbase: 1.2            # Wheelbase [m]
  control_rate: 10           # Control loop rate [Hz]